api/src/main/java/javax/enterprise/inject/spi/BeanManager.java

```diff
@@ -634,4 +636,25 @@
      */
     Event<Object> getEvent();
 
+
+    /**
+     *
+     * Obtains an {@link Instance} giving access to beans instances.
+     *
+     * The returned Instance object can only access instances of  beans that are available for injection in the module
+     * or library containing the class into which the <tt>BeanManager</tt> was injected or the Java EE component from whose JNDI
+     * environment namespace the <tt>BeanManager</tt> was obtained, according to the rules of typesafe resolution.
+     *
+     * Note that when called during invocation of an {@link AfterBeanDiscovery} event observer,
+     * this method will only return instances of beans discovered by the container before the {@link AfterBeanDiscovery} event is fired.
+     *
+     * Instances of dependent scoped beans obtained with this Instance should be explicitly destroyed with {@link Instance#destroy(Object)}
+     *
+     * @return an {@link Instance} object to request beans instances
+     * @throws IllegalStateException if called during application initialization, before the {@link AfterBeanDiscovery}
+     *         event is fired.
+     * @since 2.0
+     */
+    Instance<Object> getInstance();
```

Comment:
I would stay with `get` prefix.

> getInstance() makes it look like singletons.

Not to me. If you work with `Instace<T>`, you have to be aware that it can contain 0-n beans.